### PR TITLE
Update AutoSSL.md

### PR DESCRIPTION
**Pull request by drasbeck**
Added description for which ports to open when behind a firewall/router.

### CODE REVIEW

File: 3. Installation/4. Manual Installation/Ubuntu/Snaps/AutoSSL.md [hunk 1]
```diff
@@ -131,3 +131,10 @@ Remember to restart the Caddy service:
 ```
 sudo systemctl restart snap.rocketchat-server.rocketchat-caddy
 ```
+
+## Opening ports when running Rocket.Chat server from behind router
+For Caddy to be able to work from behind a router, the following ports need to be opened between the internet and the server. This is usually achieved through router software or web-interface.
+
+- HTTP: port **80**
+- HTTPS: port **443**
+- TCP/UDP dial-up: port **53**
```
Comment (geekgonecrazy) [inline 3. Installation/4. Manual Installation/Ubuntu/Snaps/AutoSSL.md]: From everything i've seen this shouldn't be required.

**Review by geekgonecrazy — CHANGES_REQUESTED** [review]
Lets hold off on merging this until we can verify for absolute sure that port 53 is required.  

I've done this several times and never needed inbound access to port 53

**Comment by drasbeck** [comment]
I agree, this was drafted a bit prematurely. As long as I'm the only one experiencing this it should not go in the docs.

**Comment by geekgonecrazy** [comment]
@drasbeck I think we figured it out in the [#ubuntu-snap](https://demo.rocket.chat/channel/ubuntu-snap) channel.  I think we can safely drop that and get the others merged.  Thanks :+1:

**Comment by frdmn** [comment]
I guess we can close this :)

<sub><sup>(link to the relevant conversion in [#ubuntu-snap](https://demo.rocket.chat/channel/ubuntu-snap?msg=Ne9Tbfbha5crTf3bu))</sup></sub>

**Comment by geekgonecrazy** [comment]
if we re-open and remove port 53 this might still be a useful note to add

**Comment by drasbeck** [comment]
I agree, the first part contains valuable information but we should remove the part with port 53, since I needed to open this port due to another problem entirely, and thus it does not apply to everyone.

**Comment by drasbeck** [comment]
Hi, I just removed the part with port 53.

**Comment by rafaelks** [comment]
@geekgonecrazy Can you review this one again, please?

**Review by geekgonecrazy — APPROVED** [review]

**Pull request merged**